pyiron_atomistics/atomistics/master/elastic.py

```diff
@@ -28,6 +28,51 @@
     1e21 / scipy.constants.physical_constants["joule-electron volt relationship"][0]
 )
 
+def get_elastic_tensor_by_orientation(orientation, elastic_tensor):
+    """
+    Get elastic tensor in a given orientation
+    Args:
+        orientation (numpy.ndarray): 3x3 orthogonal orientation (no need to be orthonormal)
+        elastic_tensor (numpy.ndarray): 6x6 elastic tensor
+    Returns:
+        elastic_tensor in a given orientation
```

Comment:
Should document the return type & dimension.  Should be clear in this case, but it's nice to be consistent.